src/Tribe/Main.php

```diff
@@ -671,6 +682,8 @@ public function bind_implementations() {
 		tribe_register_provider( Tribe\Service_Providers\Onboarding::class );
 		tribe_register_provider( Tribe\Admin\Notice\Service_Provider::class );
 		tribe_register_provider( Tribe\Admin\Conditional_Content\Service_Provider::class );
+
```

Comment:
```suggestion
```
The blank line in between lends gravitas to the registration of the full-site editor provider, intended?

Reply:
No, and I love that you used "gravitas"